core/src/main/java/org/seedstack/seed/core/Seed.java

```diff
@@ -177,6 +203,46 @@ synchronized private void unregisterDiagnosticManager(String name) {
         diagnosticManagers.remove(name);
     }
 
+    private String getBanner() {
+        InputStream bannerStream = SeedReflectionUtils.findMostCompleteClassLoader(Seed.class).getResourceAsStream("banner.txt");
+        if (bannerStream != null) {
+            try {
+                return new Scanner(bannerStream).useDelimiter("\\Z").next();
+            } finally {
+                try {
+                    bannerStream.close();
+                } catch (IOException e) {
+                    // nothing to do
+                }
+            }
+        }
+
+        return null;
+    }
+
+    private void checkConsistency(SeedRuntime seedRuntime) {
+        Set<String> inconsistentPlugins = new HashSet<String>();
+        String seedVersion = seedRuntime.getVersion();
+
+        if (seedVersion != null) {
+            for (Plugin plugin : ServiceLoader.load(Plugin.class)) {
```

Comment:
It would be better yes and I initially used the kernel.plugins() method but it seems that the plugins are loaded only on kernel initialization where almost all consistency errors occurs. As such the warning cannot be displayed before init without loading the plugins twice. 

Maybe we could tune the kernel to instantiate the plugins in the constructor although introducing a new phase like `prepare()` could be better.